### PR TITLE
Added spectrum comparison by using import spectrum data from CSV file 

### PR DESCRIPTION
**Pull request by demvlad**
Added comparison of two different spectrum on spectrum analyser by using import spectrum data from CSV file.
1. Plot spectrum what you want compare next time
2. Export data to CSV by pressing "Export to CSV" button.
3. Plot second spectrum
4. Press "Import from CSV" button to compare two spectrum at the chart
SpectrumComprasion

Notice:
Use same scale (min-max values)  for curves what you want to compare.

### CODE REVIEW

**Comment by nerdCopter** [comment]
personally i feel comparing two logs internally without needing to export/import would be more functional; however, i do understand such would be a huge undertaking to code.

**Comment by demvlad** [comment]
> * personally i feel comparing two logs internally without needing to export/import would be more functional; however, i do understand such would be a huge undertaking to code.
> * i suppose my idea would not work for different files, only same file, different logs :(

1. It is possible to compare different logs by using this tool
2. I am going to do possible comparing several spectrum at the one chart in the next improvement

**Review by nerdCopter — APPROVED** [review]
- approving for functionality
- request code review form others.

**Comment by demvlad** [comment]
> tested my logs (screencap). is the cyan color a good color for visibility? ![image]

Yes, I agree. I will experiment more

File: index.html [hunk 1]
```diff
@@ -475,6 +475,10 @@ <h4>Workspace</h4>
                         <div id="spectrumExport" data-toggle="tooltip" title="Export spectrum to CSV">
                             <button id="btn-spectrum-export" type="button">Export to CSV</button>
                         </div>
+                        <div id="spectrumImport"  data-toggle="tooltip" title="Import spectrum from Csv">
```
Comment (haslinghuis) [inline index.html:478]: Deja vu
```suggestion
                        <div id="spectrumImport" data-toggle="tooltip" title="Import spectrum from CSV">
```

**Comment by demvlad** [comment]
> tested my logs (screencap). is the cyan color a good color for visibility? 

I've changed color on blue. It seems, looks better.

**Comment by haslinghuis** [comment]
Pls update screenshots :)

**Comment by nerdCopter** [comment]
blue, for reviewers
image

**Review by haslinghuis — APPROVED** [review]
LGTM

**Comment by demvlad** [comment]
> * re-approving.
> * when the analyzer/window is very small, the buttons do not flow well, but i dont think it is a blocking factor.  just expand the window and they are clickable.

Yes, i know about it. Will resolve on the next step. Because delete curves action will added.

**Pull request merged**